deebot_client/messages/xml/charge_state.py

```diff
@@ -37,16 +37,20 @@ def _parse_xml(cls, event_bus: EventBus, xml: Element) -> HandlingResult:
         if (charge := xml.find("charge")) is not None and (
             charge_type := charge.attrib["type"]
         ) is not None:
+            status: None | State = None
             match charge_type.lower():
                 case "slotcharging" | "slot_charging" | "wirecharging":
                     status = State.DOCKED
                 case "idle":
-                    status = State.IDLE
+                    # Bot reports IDLE while not on the charger (e.g. while cleaning)
+                    # We ignore this state since it will conflict with the actual cleaning state
+                    pass
```

Comment:
What is the status if the bot is cleaning and you are pausing it via the app?
Newer ones will report idle there and we should do it also for the older ones if possible

Reply:
If pausing:
- PAUSED
If i stop cleaning without return to base:
- IDLE
If i press return to base
- RETURNING
Once on the base:
- DOCKED

Reply:
> If i stop cleaning without pressing return to base:
>
>    IDLE


Can we still mark the bot as idle with the above changes?